website_mrp_bom/controllers/main.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from odoo import http
+from odoo.addons.http_routing.models.ir_http import unslug
+from odoo.http import request
+
+
+class WebsiteProductPage(http.Controller):
+
+    # Do not use semantic controller due to SUPERUSER_ID
+    @http.route(['/mrp_bom/<bom_id>'], type='http', auth="public",
```

Comment:
FYI you can use <model("mrp.bom"):bom> to automatically get the bom record (without having to browse it).